sherpa/models/model.py

```diff
@@ -1362,7 +1362,7 @@ def guess(self, dep, *args, **kwargs):
 #
 def _make_unop(op: Callable,
                opstr: str,
-               strformat: Optional[str] = None) -> Callable:
+               strformat: str | None = None) -> Callable:
```

Comment:
I don't trust codecov, as I've seen pieces of code that I know are tested (e.g. I put an `assert False` in them and the tests fail) but it still claims it is not covered by the tests. Is this one of them?

This is for line 1372 but GitHub won't let me add the comment because apparently "Line must be part of the diff" so ya booh sucks to GH...

![image](https://github.com/user-attachments/assets/7ab540f2-b427-4639-9f9f-02e549c7f6c9)

Reply:
Hmm. I tried a run with this PR and

```
% git diff .
diff --git a/sherpa/models/model.py b/sherpa/models/model.py
index 94b4d593..2e6243c2 100644
--- a/sherpa/models/model.py
+++ b/sherpa/models/model.py
@@ -1368,6 +1368,7 @@ def _make_unop(op: Callable,
         def func(self):
             return UnaryOpModel(self, op, opstr)
     else:
+        assert False
         def func(self):
             return UnaryOpModel(self, op, opstr, strformat=strformat)
 
```

and the tests passed, so it does look like we don't have a test of this option (but this was with a build without XSPEC, although we should be able to test this without XSPEC models).